google-cloud-storage/src/main/java/com/google/cloud/storage/BackwardCompatibilityUtils.java

```diff
@@ -133,7 +134,6 @@ static int grpcCodeToHttpStatusCode(Code code) {
       case ABORTED: // ?
       case CANCELLED: // ?
       case UNKNOWN: // ?
-      case DATA_LOSS: // ?
```

Comment:
Are the `?` TODO's that need to be finished before cutting a relaese?

Reply:
Hopefully, they're being tracked in the tracker. However, `ABORTED`, `CANCELLED`, `UNKNOWN` probably fall into the "connection errors" bucket we've got for http.